lib/hx509/version-script.mapin

```diff
@@ -131,7 +129,6 @@ HEIMDAL_X509_1.2 {
 		hx509_certs_free;
 		hx509_certs_info;
 		hx509_certs_init;
-		hx509_certs_iter;
```

Comment:
I suppose this (and a few others) could be `#ifdef __BLOCKS__`...

Reply:
Thanks, fixed.